roles/openshift_serviceaccounts/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+- name: Create service account configs
+  template:
+    src: serviceaccount.j2
+    dest: "/tmp/{{ item }}-serviceaccount.yaml"
+  with_items: accounts
+
+- name: Create {{ item }} service account
+  command: >
+    {{ openshift.common.client_binary }} create -f "/tmp/{{ item }}-serviceaccount.yaml"
+  with_items: accounts
+
+- name: Get current security context constraints
+  shell: oc get scc privileged -o yaml > /tmp/scc.yaml
```

Comment:
Stick with {{ openshift.common.client_binary }} everywhere that you're calling oc.